lib/puppet/defaults.rb

```diff
@@ -1464,7 +1464,21 @@ def self.valid_file_checksum_types
         with the running configuration.  In the case of puppet master,
         this file reflects the state discovered through interacting
         with clients."
-      },
+    },
+    :statettl => {
+      :default => "7d",
+      :type    => :ttl,
+      :desc    => "How long the Puppet agent should cache when a resource was last checked or synced.
+      environment.
+      #{AS_DURATION}
+      A value of `0`, or `unlimited` will disable cache pruning.
+
+      This setting affects the usage of Schedule resources, as the information
```

Comment:
if you're talking about the schedule type, this should probably be lowercase and in backticks `schedule`.